packages/mobile/src/identity/actions.ts

```diff
@@ -226,6 +236,7 @@ export type ActionTypes =
   | UpdateVerificationState
   | ResendAttestations
   | SetLastRevealAttempt
+  | ReportRevealStatus
```

Comment:
Nit these interfaces typically end in `Action`, though it seems a few others in this file already break that convention